.pre-commit-config.yaml

```diff
@@ -1,10 +1,26 @@
 ---
 exclude: '^(venv|\.vscode)' # regex
 repos:
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.10
+    hooks:
+      - id: remove-tabs
+        exclude_types: [makefile, binary]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "v2.4.1"
+    hooks:
+      - id: prettier
+        args: ["--print-width=135"]
+        exclude: "(charts|templates|files)"
   - repo: https://github.com/psf/black
-    rev: 21.9b0
+    rev: 21.10b0
     hooks:
       - id: black
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.29.0
+    hooks:
+      - id: pyupgrade
+        args: ["--py38-plus"]
```

Comment:
How did you land on python 3.8+?

Reply:
This is the version we use for testing the charts. We could drop this to 3.7, but 3.6 would be too far. https://endoflife.date/python

Reply:
Gotcha, yeah since it only applies to our testing setup and a sample DAG, this is good.